firebase-functions/src/main/java/com/google/firebase/functions/FirebaseFunctions.java

```diff
@@ -187,7 +193,10 @@ URL getURL(String function) {
    *
    * @param origin The origin of the local emulator, such as "http://10.0.2.2:5005".
    */
-  public void useFunctionsEmulator(String origin) {
+  public void useFunctionsEmulator(@NonNull String origin) {
+    if (origin == null) {
+      throw new IllegalArgumentException("origin cannot be null");
```

Comment:
nit: Maybe use `Preconditions.checkNotNull()` instead? Since that's the approach used in `getInstance()`

Reply:
Agreed. I'll fix this one, which I added, together with the in `call()` in a follow up PR. Thanks!